languages/tree-sitter-stack-graphs-java/rust/lib.rs

```diff
@@ -26,6 +26,8 @@ pub fn language_configuration(cancellation_flag: &dyn CancellationFlag) -> Langu
         Some(STACK_GRAPHS_BUILTINS_CONFIG),
         FileAnalyzers::new(),
         cancellation_flag,
-    )
-    .unwrap()
+    ) {
+        Ok(lc) => lc,
+        Err(err) => panic!("{}", err),
+    }
```

Comment:
I'm not sure one is better than the other, but if you'd prefer to avoid the `match` and use the `unwrap` format, this could be:

```rust
    .unwrap_or_else(|err| panic!("{}", err))
```

Reply:
That's much nicer. Thanks for the tip!